assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -233,4 +281,11 @@ function Carousel() {
   );
 }
 
+const Li = styled.li`
+  margin: 0 ${({ marginRight }) => marginRight}px 0 0;
+  &:last-of-type {
```

Comment:
Does this even make a difference? Removing it doesn't cause any changes, presumably because of flexbox. So seems we could remove that.

Reply:
I started with that, but here is the problem (not centered content), of course, right now our spacing is tiny so just 0.1% of people will notice that 😉 :

![image](https://user-images.githubusercontent.com/21036224/75693564-67dc3500-5ca7-11ea-88e5-9bff8306cd11.png)